tools/acc_forwarder/Cargo.toml

```diff
@@ -15,8 +15,8 @@ futures = "0.3.28"
 lazy_static = "1.4.0"
 log = "0.4.17"
 mpl-token-metadata = "1.8.3"
-plerkle_messenger = { version = "1.5.2", features = ["redis"] }
-plerkle_serialization = "1.5.2"
+plerkle_serialization = { git = "https://github.com/metaplex-foundation/digital-asset-validator-plugin.git", rev = "35a8801c" }
```

Comment:
You could probably just use `plerkle_serialization = { version = "1.5.2" }` since that's what we had to leave it in the ingester due to blockbuster using that version... But its fine to use the pinned version as well.